### ISSUE TITLE
nice servers valve, did you seriously punish me with ban after your own server kicked me out and didnt let me to reconnect?

### ISSUE THREAD

**Issue by s0ul0fwarri0r** [ValveSoftware/Dota2-Gameplay#15269]
### Description

AND AFTER I SPENT MONEY ON 8 GIFT TREASURES TOO BTW ;)

**Comment by WendaoLee** [comment]
Bro,I know how angry you are,but this is a channel of bug tracker.....so your posts here were useless. :(

**Comment by DocMorg** [comment]
same here, wanted to thx valve for bans

**Comment by s0ul0fwarri0r** [comment]
> same here, wanted to thx valve for bans

even though i didnt get banned, only server didnt let me in multiple times and people keep crashing during the matches i played

**Comment by s0ul0fwarri0r** [comment]
> Bro,I know how angry you are,but this is a channel of bug tracker.....so your posts here were useless. :(

yeah, i guessed as much, even though its as obvious as a bug as when i try to buy cards to level up my badges and have to click many many many times to even be allowed to buy them.
